multimodal/src/autogluon/multimodal/models/mmocr_text_recognition.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import warnings
+from typing import List, Optional
+
+import torch
+from mim.commands.download import download
+from torch import nn
+
+try:
+    import mmcv
+    from mmcv.parallel import scatter
+    from mmcv.runner import load_checkpoint
+except ImportError:
+    mmcv = None
+
+try:
+    import mmocr
+    from mmocr.models import build_detector
+    from mmocr.utils.model import revert_sync_batchnorm
+except ImportError:
+    mmocr = None
+
+try:
+    import mmdet
+    from mmdet.core import get_classes
+except ImportError:
+    mmdet = None
```

Comment:
How about moving this import block to `utils.py`? It seems that we don't need them here.

Reply:
Also, the same import block exists in `mmocr_text_detection`, which we may remove.

Reply:
I will move them to the get_mmocr_models function in utils.py.